lib/gooddata/rest/connection.rb

```diff
@@ -466,6 +466,17 @@ def generate_request_id
         "#{@execution_id}:#{session_id}:#{call_id}"
       end
 
+      def enrich_error_message(exception)
+        begin
+          response = JSON.parse(exception.response.body, symbolize_names: true)
+          exception.define_singleton_method :to_s do
+            super() + ': ' + response[:error][:message] % response[:error][:parameters] + 'request_id: ' + response[:error][:requestId]
```

Comment:
Can we rely on `response[:error][:message]` being in the response? It might be a good idea to catch possible failures.